src/app/components/core/developer/additional-services/additional-services-utils.service.ts

```diff
@@ -83,7 +83,7 @@ export class AdditionalServicesUtils {
         (data: any) => {
           let servicesRegistry = data.services.filter((el) => expectedExternalServiceType.includes(el.type))
           if (from !== 'additional-services') {
-            servicesRegistry = data.services.filter((s => (s.type.toLowerCase() === from)));
+            servicesRegistry = data.services.filter((s => (s.type.toLowerCase() === (from === 'bucket' ? 'bucketstorage' : from))));
```

Comment:
We better have clarity in statements as

const origin = from === 'bucket' ? 'bucketstorage' : from;
const servicesRegistry = data.services.filter(service => service.type.toLowerCase() === origin);